### PR TITLE
Use US English and fix a few more typos

### PR DESCRIPTION
**Pull request by veger**
Part II of #73

### CODE REVIEW

File: YAFC/Workspace/ProductionTable/ProductionTableView.cs [hunk 1]
```diff
@@ -298,22 +298,22 @@ private class EntityColumn : ProductionTableDataColumn {
 
             private void ShowAccumulatorDropdown(ImGui gui, RecipeRow recipe, Entity accumulator) {
                 gui.ShowDropDown(imGui => {
-                    imGui.BuildInlineObjectListAndButton<EntityAccumulator>(Database.allAccumulators, DataUtils.DefaultOrdering,
```
Comment (have-fun-was-taken) [inline YAFC/Workspace/ProductionTable/ProductionTableView.cs:301]: The type was removed there. Could you please elaborate on that?

Comment (veger) [reply YAFC/Workspace/ProductionTable/ProductionTableView.cs:301]: Not really... :smile:
I use(d) the formatting tools to rename variables/functions, so I didn't notice.
Let me check, maybe it became/is obsolete and by refactoring that line other tools removed it automatically? :thinking:

Comment (veger) [reply YAFC/Workspace/ProductionTable/ProductionTableView.cs:301]: Yes, it was indeed something that got fixed by accident: [Simplify name (IDE0001)](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0001?pivots=lang-csharp-vb)

I'll revert to keep this PR clean (there are lots of other fixes pointed out by editorconfig to be made :stuck_out_tongue: )

Comment (veger) [reply YAFC/Workspace/ProductionTable/ProductionTableView.cs:301]: I use the 'rename' functionality to fix the names of the variables/functions. This one got changed as a 'dependency'. I guess it triggered the 'full cleanup' on that line or something?

Comment (veger) [reply YAFC/Workspace/ProductionTable/ProductionTableView.cs:301]: As far as I know there is no global clean up. For this PR I just opened all code/MD files and looked if there were cSpell warnings to fix.
image

Comment (have-fun-was-taken) [reply YAFC/Workspace/ProductionTable/ProductionTableView.cs:301]: Regarding the revert that you did, the rest of that function also got changed in the fix, but it was not reverted.

Comment (veger) [reply YAFC/Workspace/ProductionTable/ProductionTableView.cs:301]: Oh... I didn't notice :disappointed:

Comment (veger) [reply YAFC/Workspace/ProductionTable/ProductionTableView.cs:301]: FYI After reverting, I used 'rename' to rename `accum` -> `accumulator` again and it triggered the other cleanups. 
As a matter of fact it was impossible to use the rename without triggering and I changed both manually.

So it looks at surrounding code and triggers the cleanup? :thinking:

Comment (have-fun-was-taken) [reply YAFC/Workspace/ProductionTable/ProductionTableView.cs:301]: Interesting. I'm not against that change, just wanted to know why exactly that function was changed beyond the rename.

Comment (have-fun-was-taken) [reply YAFC/Workspace/ProductionTable/ProductionTableView.cs:301]: Sure, if you feel like doing that, I'll be happy to help with the review. But this is optional.

Comment (veger) [reply YAFC/Workspace/ProductionTable/ProductionTableView.cs:301]: Then let's leave it :smile: 
I don't feel too confident what is a right change and what is not. For the grammar/typos/US vs UK EN I got triggered. The other cleanups a whole lot less.

**Review by have-fun-was-taken — APPROVED** [review]

**Comment by have-fun-was-taken** [comment]
Feel free to merge when you're ready.

**Pull request merged**